__tests__/modules/generateWallet.test.js

```diff
@@ -29,31 +35,183 @@ describe('generateWallet module tests', () => {
 
   describe('newWallet tests', () => {
     const payload = account
-    const expectedAction = Object.assign({}, { payload }, { type: NEW_WALLET })
+    const expectedAction = Object.assign({}, { payload }, { type: NEW_WALLET_ACCOUNT })
```

Comment:
Do we need an Object.assign here?
Can it be - 
`const expectedAction = { payload, type: NEW_WALLET_ACCOUNT }`